apps/client/app/employee/[id]/report/[cycleId]/page.tsx

```diff
@@ -1,47 +1,95 @@
 import axios from 'axios'
 import '../../../../global.css'
-import { getReport, getUser } from '@/lib/fetch'
 
-
-const userQuery = `
-query getUser($id: String) {
-  getUser(id:$id) {
-    fullName
-}
-}`
-
-const reportQuery = `
-query GetReport($targetId: String!, $cycleId: String!) {
-  getReport(targetId: $targetId, cycleId: $cycleId) {
-    _id {
-      target
-      cycle
-    }
-    remarks
-  }
-}`
+import {
+  Table,
+  TableBody,
+  TableCaption,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table"
+import { getReport, getUser, getFullReport } from '@/lib/fetch'
+import { userQuery, reportQuery, getFullReportQuery } from '@/lib/queries'
 
 
 async function Report({ params }) {
+  const manager = true
+
   const targetId = params.id
   const cycleId = params.cycleId
 
   // const status = undefined
   const variables = { targetId, cycleId }
 
+
+  const user = await getUser(userQuery, params)
```

Comment:
pls remove the `getUser` and `getReport` if you're not using them

if you're going to use the `user`, need to fix the `getUser` -- it's passing the wrong params right now